services/motion/builtin/builtin_test.go

```diff
@@ -1731,25 +1279,897 @@ func TestMoveOnGlobe(t *testing.T) {
 
 func TestMoveOnMapNew(t *testing.T) {
 	ctx := context.Background()
+	logger := logging.NewTestLogger(t)
 
-	base, ms := createMoveOnMapEnvironment(
-		context.Background(),
-		t,
-		"slam/example_cartographer_outputs/viam-office-02-22-3/pointcloud/pointcloud_4.pcd",
-		110,
-		nil,
-	)
-	defer ms.Close(ctx)
+	t.Run("Long distance", func(t *testing.T) {
+		if runtime.GOARCH == "arm" {
```

Comment:
Will this also skip tests on M1 macs? If so I think we need to be more spectific than this.

Reply:
this does not skip on M1 macs.

Reply:
In go-world, `arm` refers specifically to 32-bit arm. an M1 will report as "arm64".